cli/tests/pcluster/schemas/test_imagebuilder_schema.py

```diff
@@ -47,30 +49,64 @@
                     }
                 ],
             },
+            None,
+        ),
+        (
+            "imagebuilder_schema_dev.yaml",
```

Comment:
Same as above. Having test case ids it's helpful to understand also the reason why we are adding new test cases. In this case, why was it necessary? What's the unknown field?